src/lightning_app/runners/cloud.py

```diff
@@ -418,9 +418,11 @@ def dispatch(
                     initial_port += 1
 
             queue_server_type = V1QueueServerType.UNSPECIFIED
-            if CLOUD_QUEUE_TYPE == "http":
+            # Note: Enable app to select their own queue type.
+            queue_type = get_cloud_queue_type()
```

Comment:
don't you want to put this logic directly into `get_cloud_queue_type`, so that it returns a V1QueueServerType? Seems like the perfect opportunity for a clean up

Reply:
This function is used in other places as a string. Let's maybe refactor this in another PR.